spec/requests/proxying_spec.rb

```diff
@@ -2,8 +2,17 @@
 
 RSpec.describe "Proxying requests", type: :request do
   let(:body) { "abc" }
-  let(:upstream_path) { "/foo" }
-  let(:upstream_uri) { ENV['GOVUK_UPSTREAM_URI'] }
+  let(:upstream_slug) { "/foo" }
+  let(:upstream_path) { upstream_slug }
+  let(:upstream_domain) { ENV['GOVUK_UPSTREAM_URI'] }
+  let(:upstream_uri) { upstream_domain + upstream_path }
+  let(:authenticated_user_uid) { User.first.uid }
+  let(:authenticated_org_content_id) { User.first.organisation_content_id }
+  let(:auth_bypass_id) { SecureRandom.uuid }
+  let(:jwt_auth_secret) { 'my$ecretK3y' }
+  let(:valid_token) { JWT.encode({ 'sub' => auth_bypass_id }, jwt_auth_secret, 'HS256') }
+  let(:invalid_token) { JWT.encode({ 'sub' => auth_bypass_id }, 'invalid', 'HS256') }
+  let(:valid_token_with_wrong_key) { JWT.encode({ 'foo' => 'bar' }, 'invalid', 'HS256') }
```

Comment:
Looks like we've built up quite a collection of lets here. Do we need them all to be global? having them all here is a bit of a jump from reading the test context so I'd suggest seeing if they can be more contextual. If a couple have to be duplicated it's not too big a deal as it'd likely be an easier reading experience.